### PR TITLE
Update UG for View and List commands

### PR DESCRIPTION
**Pull request by sudarshan2401**
Updated command parameters for View Command and added List Command to UG

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -267,5 +273,5 @@ Coming soon...
...
 | **add**    | **Adding a class:** <br> **Format:** `add /c CLASS_NAME` <br> **Eg:** `add /c CS2103T` <br> <br> **Adding a student:** <br> **Format:** `add /s STUDENT_NAME  /c CLASS_NAME` <br> **Eg:** `add /s John /c CS2103T`  <br><br> **Adding a list of students** <br> Coming soon |
 | **remove** | **Removing a class:** <br> **Format:** `remove /c CLASS_NAME` <br> **Eg:** `remove /c CS2103T` <br> <br> **Removing a student:** <br> **Format:** `remove /s STUDENT_NAME  /c CLASS_NAME` <br> **Eg:** `remove /s John /c CS2103T`                                          |
-| **view**   | **Viewing a class:** <br> **Format:** `view /c CLASS_NAME` <br> **Eg:** `view CS2103T`                                                                                                                                                                                      |
+| **view**   | **Viewing a class:** <br> **Format:** `view /c CLASS_INDEX` <br> **Eg:** `view 1`                                                                                                                                                                                           |
```
Comment (songgthu) [inline docs/UserGuide.md]: There's a small typo here, should be `view /c 1` in the E.g part

**Review by songgthu — APPROVED** [review]
LGTM! Just 1 small typo that needs to be fixed.

File: docs/UserGuide.md (hunk 1 above)
Comment (sudarshan2401) [reply docs/UserGuide.md]: Thanks, my bad!

**Review by revdrag — APPROVED** [review]
LGTM!

**Pull request merged**